cibuildwheel/windows.py

```diff
@@ -317,20 +317,22 @@ def build(options: BuildOptions) -> None:
                 config_setting = " ".join(verbosity_flags)
                 build_env = env.copy()
                 if options.dependency_constraints:
-                    constr = options.dependency_constraints.get_for_python_version(config.version)
+                    constraints_path = options.dependency_constraints.get_for_python_version(
+                        config.version
+                    )
```

Comment:
FYI, this originally was exactly the number of characters that kept this from wrapping. ;) This is fine, though.